recipe/meta.yaml

```diff
@@ -7,47 +7,49 @@ package:
 source:
   fn: sunpy-{{version}}.tar.gz
   url: https://pypi.io/packages/source/s/sunpy/sunpy-{{ version }}.tar.gz
-  sha256: c14b6be4f4d705a235c26b1daa6dfb8136185e19ed1dc2e151561f602b3befb1
+  sha256: 8d9536ac4c779b0cee7f1cde4575c2bf5655ef67d67a0d25e39aa13a85ad75f1
 
 build:
   number: 0
-  script: python setup.py install --offline --no-git --single-version-externally-managed --record record.txt
+  script: python -m pip install . --no-deps -vv
```

Comment:
I think we still need this for `astropy_helpers` reasons right?

Reply:
Maybe?